### DISCUSSION TITLE
Webapp notifications not going to notifications list

### DISCUSSION THREAD

**Discussion by htbg** [linuxmint/discussions#504] [Questions]
I currently use several webapps for microsoft teams and outlook.

The notifications work as expected, showing in the desktop.

However, they are not showing in the notifications list (standard mint panel applet).

Is there any way to make them appear there as well?

**Comment by rcalixte** [comment]
They should appear there as long as you have not interacted with the web application itself. This could be an issue with the specific applications here as I can not replicate it with other web applications. Provided that the notifications are displaying correctly, they would remain in your notification drawer with the same persistence as your other notifications unless the application itself directed otherwise. Is there a chance you have some other client also interacting with these platforms?

**Reply by htbg** [reply]
It's a strange behaviour that only seems to occur on Microsoft Teams... my Outlook webapps' notifications stay there in the trail with no issues.

About another client interacting with these platforms, how could I check that?

**Reply by rcalixte** [reply]
> About another client interacting with these platforms, how could I check that?

Simple. Do you have a mobile client on Teams? Or another computer? If you're logged in anywhere else, that could be causing issues. The fact that it works for Outlook and not Teams is interesting. Have you tried using other browsers to configure the webapp too?
